src/geometry/PolyUtil.js

```diff
@@ -54,3 +54,45 @@ L.PolyUtil.clipPolygon = function (points, bounds, round) {
 
 	return points;
 };
+
+/* @function ringContains(points: Point[], point: Point): Boolean
+ * Returns `true` if `point` is inside the ring defined by `points`.
+ * @alternative
+ * @function ringContains(latlngs: LatLng[], latlng: LatLng, map: Map): Boolean
+ * Returns `true` if the `map` projection of `latlng` is inside the ring defined by
+ * projected `latlngs`.
+ */
+L.PolyUtil.ringContains = function (objs, obj, map) {
+	var point, points;
+
+	if (objs[0] instanceof L.Point && obj instanceof L.Point) {
+		point = obj;
+		points = objs;
+	} else if (map && objs[0] instanceof L.LatLng && obj instanceof L.LatLng) {
+		point = map.project(obj);
+		points = [];
+		for (var j = 0; j < objs.length; j++) {
+			points.push(map.project(objs[j]));
+		}
+	}

```

Comment:
If this is considered a public API, we might want to raise an error about incompatible input arguments in the else clause here. Or just name the arguments `points, point`, and only have the `(map && objs[0] instanceof L.LatLng && obj instanceof L.LatLng)` check for the second case. Now it will fail with something like `points.length` is not defined, when called with `[point, point, point, ...], latlng`, which might be confusing.